integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -73,15 +73,13 @@ master:
   # JVM options specific to the master container
   jvmOptions:
   nodeSelector: {}
-  hostNetwork: false
-  dnsPolicy: ClusterFirst
```

Comment:
Removed these two by default and use inference. However they can still be set to override, especially when we want to use some dnsPolicy other than `ClusterFirst` and `ClusterFirstWithHostNet`

Reply:
instead of removing this, i would comment it out and explain the order of precedence

Reply:
Good idea!